src/staging/gatsby-config.js

```diff
@@ -98,7 +98,18 @@ module.exports = {
     {
       resolve: `gatsby-plugin-robots-txt`,
       options: {
-        policy: [{userAgent: '*', allow: '/'}]
+        resolveEnv: () => process.env.ENV_SUFFIX,
```

Comment:
can we default this to "dev" in case it's running on an environment with no variable set?
```
resolveEnv: () =>  process.env.ENV_SUFFIX || 'dev'
 ```